packages/domain/src/use-cases/wallets/WalletLogin.ts

```diff
@@ -74,8 +74,8 @@ export class WalletLogin {
       request.handle,
     );
 
-    this.activeProfilePresenter.presentActiveProfile(profile);
```

Comment:
This line here was causing some state inconsistencies inside `isOwnedByMe` which requires the wallet to be already present. This indicates also why it's important to keep that state together as it's strongly coupled together to some extend. See https://avara.height.app/T-12199